Base/res/fortunes.json

```diff
@@ -35,36 +35,36 @@
     "quote": "We can't not have the quotes under VC, that's a sin",
     "author": "CxByte",
     "utc_time": 1612035713,
-    "url": "https://freenode.logbot.info/serenityos/20210130#c6726388",
+    "url": "https://benwiederhake.github.io/freenode-serenity-archive/quote-122663.html",
     "context": "A quote about putting quotes in VC, in VC."
   },
   {
     "quote": "\"We really should start a \\\"Quotes\\\" page.\" - BenW",
     "author": "kling",
     "utc_time": 1612900961,
-    "url": "https://freenode.logbot.info/serenityos/20210209#c6854420",
+    "url": "https://benwiederhake.github.io/freenode-serenity-archive/quote-125770.html",
     "context": "Apparently I said that once too often."
   },
   {
     "quote": "if your port uses textrels, it likely uses other things that we don't want near serenity",
     "author": "thakis",
     "utc_time": 1612900918,
-    "url": "https://freenode.logbot.info/serenityos/20210209#c6854410",
+    "url": "https://benwiederhake.github.io/freenode-serenity-archive/quote-125768.html",
     "context": "\"jk but only a little bit jk\""
   },
   {
     "quote": "I'm afraid of where pulling that string will take me.",
     "author": "boricj",
     "utc_time": 1612954860,
-    "url": "https://freenode.logbot.info/serenityos/20210210#c6866141",
+    "url": "https://benwiederhake.github.io/freenode-serenity-archive/quote-125894.html",
     "context": "C++ templates will lead you down the rabbithole."
   },
   {
     "quote": "$commitdescription ($user opened: The build failed.)",
     "author": "SerenityBot",
     "utc_time": 1613906220,
     "url": "https://freenode.logbot.info/serenityos/20210221#c6989891",
-    "context": "The IRC notifications are a little bit harsh sometimes, especially if they all seem to spell failure."
+    "context": "https://benwiederhake.github.io/freenode-serenity-archive/quote-128669.html"
```

Comment:
I think you meant to update the `url` key here?

Reply:
Derp! Yes, you're right.